templates/types/streaming/fastapi/app/engine/index.py

```diff
@@ -0,0 +1,16 @@
+import logging
+import os
+
+from llama_index.core.indices import VectorStoreIndex
+from app.engine.vectordb import get_vector_store
+
+
+logger = logging.getLogger("uvicorn")
+
+
+def get_index():
+    logger.info("Connecting vector store...")
+    store = get_vector_store()
+    index = VectorStoreIndex.from_vector_store(store)
```

Comment:
is this using the doc_store?

Reply:
No, we are only using the doc_store for deduplication in generating the index.

Reply:
yes but the doc_store should also be used if the index is read